mods/screwdriver/init.lua

```diff
@@ -66,7 +66,7 @@ screwdriver.handler = function(itemstack, user, pointed_thing, mode, uses)
 		if not ndef or not ndef.paramtype2 == "facedir" or
 				ndef.on_rotate == false or
 				(ndef.drawtype == "nodebox" and
-				not ndef.node_box.type == "fixed") or
+				(ndef.node_box and not ndef.node_box.type == "fixed")) or
```

Comment:
Could you please also change it to `and ndef.node_box.type ~= "fixed"` for better readability?
Otherwise :+1: